### PR TITLE
feat(video): add component

### PR DESCRIPTION
**Pull request by ofekashery**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

## Change information
Added new video component
image

### CODE REVIEW

File: components/video/video.tsx [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+              <div className="video-controls">
+                {isPlaying ? (
+                  <button onClick={pause} className="play-button">
+                    <PauseIcon size={14} />
+                  </button>
+                ) : (
+                  <button onClick={play} className="play-button">
+                    <PlayIcon size={14} />
+                  </button>
+                )}
```
Comment (unix) [inline components/video/video.tsx:157]: Maybe we can abstract this into a single component? The `video.tsx` file looks a little big.

```jsx
<VideoControlButton isPlaying={isPlaying} onPause={pause} onPlay={play} />
<VideoControlFullScreen isFullScreen={isFullScreen} onClick={fullScreenClick} />
```

**Comment by unix** [comment]
We can use  the `slider` component, but force changes to its style? Currently it seems that the implemention of drag and drop uses a lot of code.

What's more, we need to rewrite a lot of testcase about drag and drop. 😿 

```jsx
const VideoProgress = () => {
  return (
    <div className="video-progress">
      <Silder {...} >
      <style jsx>{`
        .video-progress :global(.something) { ... }
      `}</style>
    </div>
  )
}
```

**Comment by ofekashery** [comment]
It looks like we need a combination of ``Progress`` and ``Slider``.

**Comment by unix** [comment]
> It looks like we need a combination of `Progress` and `Slider`.

Yes, I found out.

How about setting1 `.slider background` to `transparent` and putting it on top? We get the change of value from `onChange`, then set to your progress element?

**Pull request closed without merging**